third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -4356,6 +4356,71 @@ def fillna(self, value):
         """
         raise NotImplementedError(constants.ABSTRACT_METHOD_ERROR_MESSAGE)
 
+    def replace(
+        self,
+        to_replace,
+        value=None,
+        *,
+        regex=False,
+    ):
+        """
+        Replace values given in `to_replace` with `value`.
+
+        Values of the Series/DataFrame are replaced with other values dynamically.
+        This differs from updating with ``.loc`` or ``.iloc``, which require
+        you to specify a location to update with some value.
+
+        Args:
+            to_replace (str, regex, list, int, float or None):
+                How to find the values that will be replaced.
+
+                * numeric, str or regex:
```

Comment:
I don't think the bulletpoint will be rendered correctly, we should get rid of it. Could you help do a staging following the instructions: https://screenshot.googleplex.com/5MQFPth8968Qjtw.

Just like: https://screenshot.googleplex.com/BPQxVeg7D7NjjTZ

Reply:
removed bullet points

Reply:
Not sure how the CloudRAD tool will render this. Will LGTM for this one, and we can check back if we need to re-format it.